release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2966,6 +2966,16 @@ For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-d
 +
 If these conditions are not met, the pod can fail at admission with `UnexpectedAdmissionError` error, which is an expected behavior. If the application pod is part of deployments, in case of failure subsequent pods are spun up and ultimately successfully run when devices are suitable to be allocated. (link:https://issues.redhat.com/browse/OCPBUGS-14438[*OCPBUGS-14438*])
 
+* Previously, because client TLS (mTLS) was configured on an ingress controller, and the certificate authority (CA) in the client CA bundle required more than 1MB of certificate revocation list (CRLs) to be downloaded, the CRL `ConfigMap` object could not be updated due to the size limitation on `ConfigMaps`. As a result of the missing CRLs, connections with valid client certificates may have been rejected with the error `unknown ca`. With this update, the CRL `ConfigMap` for each ingress controller no longer exists; instead, CRLs are downloaded directly by each router pod ensuring connections with valid client certificates are no longer rejected. (link:https://issues.redhat.com/browse/OCPBUGS-13967[*OCPBUGS-13967*])
+
+* Previously, because client TLS (mTLS) was configured on an ingress controller, mismatches between the distributing certificate authority (CA) and the issuing CA caused the incorrect certificate revocation list (CRL) to be downloaded. As a result, the incorrect CRL would be downloaded in place of the correct CRL causing connections with valid client certificates to be rejected with the error message `unknown ca`. With this update, downloaded CRLs are now tracked by the CA that distributes them; rather than, their issuing CA. This ensures that valid client certificates are no longer rejected. (link:https://issues.redhat.com/browse/OCPBUGS-13964[*OCPBUGS-13964*])
+
+
+Cause: When client TLS (mTLS) was configured on an ingress controller, if any of the client CA certificates included a CRL distribution point for a CRL issued by a different CA and that CRL expired, the mismatch between the distributing CA and the issuing CA caused the incorrect CRL to be downloaded.
+Consequence: The CRL bundle would be updated to contain an extra copy of the erroneously downloaded CRL, and the CRL that needed to be updated would be missing entirely. Because of the missing CRL, connections with valid client certificates may have been rejected with the error "unknown ca"
+Fix: Downloaded CRLs are now tracked by the CA that distributes them, rather than their issuing CA
+Result: When a CRL expires, the distributing CA's CRL distribution point is used to download an updated CRL. Valid client certificates will no longer be rejected.
+
```

Comment:
Is this intended to be here? It looks like the unedited version of the OCPBUGS-13964 note.

Reply:
Ummm, nothing to see here just my unusual workflow. 😆 thanks for the catch Ryan.